src/main/java/com/metamx/common/StreamUtils.java

```diff
@@ -35,6 +37,7 @@
   // The default buffer size to use (from IOUtils)
   private static final int DEFAULT_BUFFER_SIZE = 1024 * 4;
 
+  @Deprecated // Use FileUtils.retryCopy
```

Comment:
FileUtils.retryCopy only takes a ByteSource, it cannot work on InputStream directly, so we cannot deprecate it if we don't offer the same functionality.